README.md

```diff
@@ -88,3 +88,55 @@ Visit our documentation on [readthedocs.org](http://ralph-ng.readthedocs.org)
 ## Getting help
 
 Ralph community will answer your questions on a chat: [![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/allegro/ralph?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+
+# Dev Tips & Tricks
+
+## how to see asset boxes in front dashboard
+
+By default, when you visit front dashboard, you see no "Hardware loan", "Hardware release" nor "Hardware return" boxes. They should be visible only when you have assigned asset in back office, which is in 'in progress' or 'in use' state. However - when you add such item on empty database, you will ecounter error: 
+
+```
+NoReverseMatch at /
```

Comment:
I admit, it would be awesome. However, sounds like "big thing" and subject of separate issue. 

This too complicated instruction is better than WTF feeling when you ecounter "NoReverseMatch" :).